### DISCUSSION TITLE
How to exclude a RESX file?

### DISCUSSION THREAD

**Discussion by LibreShawn** [dotnet/ResXResourceManager#620] [Q&A] [answered]
Is it possible to exclude a RESX file from all processing done by this extension. I have about 30 language based Strings.??.resx files and one Common.resx file. The Common.resx file contains strings that shouldn't be translated, like names and acronyms. I would like this file not to show up in the manager and I really don't want to see translation warnings for this file.

**Comment by tom-englert** [comment, marked as answer]
Just mark all entries as invariant
There is also a regex in the configuration to exclude files

**Reply by LibreShawn** [reply]
Thank you so much for the prompt and useful reply. Both methods did the trick for me. I went with the regex because it doesn't show the file in the extension at all. Perfect!
